projects/ng-animate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-animate",
-  "version": "1.0.0",
+  "version": "14.0.0",
```

Comment:
Is it some convention to have an angular library version match with Angular? Otherwise I'd prefer just bumping to 2.0

Reply:
yeah makes sense, I usually keep it in sync with ng to avoid confusion but as long as it's marked as breaking change it's good to go.

v2